src/lib/ngsi10/UpdateContextResponse.cpp

```diff
@@ -163,3 +164,96 @@ void UpdateContextResponse::release(void)
   contextElementResponseVector.release();
   errorCode.release();
 }
+
+
+
+/* ****************************************************************************
+*
+* UpdateContextResponse::notFoundPush - 
+*
+* 1. Find contextElementResponse in contextElementResponseVector and add the ContextAttribute.
+* 2. If not found: create a new one.
+*
+*/
+void UpdateContextResponse::notFoundPush(EntityId* eP, ContextAttribute* aP)
+{
+  ContextElementResponse* cerP = contextElementResponseVector.lookup(eP, SccContextElementNotFound);
+  if (cerP == NULL)
+  {
+    cerP = new ContextElementResponse(eP, aP);
+    cerP->statusCode.fill(SccContextElementNotFound);
+    contextElementResponseVector.push_back(cerP);
+  }
+  else
+  {
+    cerP->contextElement.contextAttributeVector.push_back(aP);
+  }
+}
+
+
+
+/* ****************************************************************************
+*
+* UpdateContextResponse::foundPush - 
+*
+* 1. Find contextElementResponse in contextElementResponseVector and add the ContextAttribute.
+* 2. If not found: create a new one.
```

Comment:
Not sure if this comment (bullet 2) is correct, taking into account the name of the function...